### ISSUE TITLE
how to control the row numbers in view, when use nzVirtual? 虚拟滚动，如何控制默认显示多少行

### ISSUE THREAD

**Issue by XimenaWang** [NG-ZORRO/ng-zorro-antd#8441]
## What problem does this feature solve?
设置表格为虚拟滚动，但是设置 scroll的 y值不生效；表格底部横线与 X滚动条未重合，中间有很大空白区域

## What does the proposed API look like?
nzVirtualHeight

**Comment by XimenaWang** [comment]
hi zorro-bot, I tried this API 'nzVirtualHeight', but it dosen't work,

**Comment by Laffery** [comment]
Hi, 请参考官网文档给出的示例，`nzVirtualItemSize` 为 item 高度，`nzScroll.y` 为虚拟滚动视口高度，默认显示行数 = `nzScroll.y / nzVirtualItemSize`。

```html
<nz-table
  #virtualTable
  [nzBordered]="true"
  [nzVirtualItemSize]="54"
  [nzData]="listOfData"
  [nzVirtualForTrackBy]="trackByIndex"
  [nzFrontPagination]="false"
  [nzShowPagination]="false"
  [nzScroll]="{ x: '1200px', y: '240px' }"
></nz-table>
```

若仍不能解决您的问题，请提供问题复现链接

**Comment by XimenaWang** [comment]
@Laffery 是的, 按照官网连接， nzScroll.y 是控制显示行数的，但是很奇怪，比如我实际有30行数据，每一行高度nzVirtualItemSize =42, 总高度设置为6行的高度：252，就显示正常，底部的滚动条紧挨着表格底部；

但是如果我设置nzScroll.y大于300， 就会表格和底部滚动条有一段距离，像下图
![image](https://github.com/NG-ZORRO/ng-zorro-antd/assets/30890958/60516b3c-0f4e-4a6a-a8cb-7f7b016eb16f)
